packages/back-end/src/models/OrganizationModel.ts

```diff
@@ -277,3 +295,18 @@ export async function findOrganizationsByDomain(domain: string) {
   const docs = await OrganizationModel.find({ verifiedDomain: domain });
   return docs.map(toInterface);
 }
+
+export async function setOrganizationMessages(
+  orgId: string,
+  messages: OrganizationMessage[]
+): Promise<void> {
+  await OrganizationModel.updateOne(
+    {
+      id: orgId,
+    },
+    { messages },
+    {
+      runValidators: true,
```

Comment:
This is really bizarre but for `update` methods, validators are disabled by default for Mongoose ([source](https://mongoosejs.com/docs/validation.html#update-validators)).